assets/aml-benchmark/components/prompt_crafter/spec.yaml

```diff
@@ -0,0 +1,115 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: prompt_crafter_convergence_test
```

Comment:
The name of the component does not need `convergence_test`.

Reply:
updated. Thanks!